pilot/pkg/proxy/envoy/v2/debug_test.go

```diff
@@ -467,15 +467,21 @@ func TestEvaluateTLSState(t *testing.T) {
 }
 
 func TestAnalyzeMTLSSettings(t *testing.T) {
+	fakeConfigMeta := model.ConfigMeta{
+		Name:      "foo",
+		Namespace: "bar",
+	}
 	testCases := []struct {
```

Comment:
does this work with mesh policy?

ideally users should be able to tell if this is from authn policy and mesh policy, right?

Reply:
Yes. mesh policy should have namespace empty, so will be shown as "default/-" in the output.

Reply:
okay, i assume our config store part has taken care of meshpolicy resolution in that case. (fill empty namespace)